src/schema/rules/files/raw/photo.yaml

```
@@ -19,9 +17,7 @@ photo:
 photo__micr:
   $ref: rules.files.raw.photo.photo
   extensions:
-    - .jpg
-    - .png
-    - .tif
+    - [.jpg, .png, .tif]
```

This will save us the trouble of updating it twice if other formats are accepted in the future.

```suggestion
    - $ref: rules.files.raw.photo.photo.extensions
```

I think the changes in @tsalo's enum values PR should allow this to just work.